Sources/TrustlySDK/Helpers/ValidationsHelper.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ValidationsHelper.swift
+//  TrustlySDK
+//
+//  Created by Marcos Rivereto on 05/07/24.
+//
+
+import Foundation
+import WebKit
+
+
+class ValidationHelper {
+    
+    static func isErrorCodePage(matches: [NSTextCheckingResult], content: String) -> Bool {
+        for match in matches {
+
+            let value = (Int(content[Range(match.range,in: content)!]) ?? 0) / 100
+
+            if value == 4 || value == 5 {
```

Comment:
Maybe is a good pattern create some constants just to explain why those values

Reply:
Fixed